src/components/FundsBanner/FundsBanner.scss

```diff
@@ -0,0 +1,35 @@
+.funds-banner {
+    @include mobile-or-tablet-screen {
+        padding: 1.6rem 1.6rem 0.8rem;
+    }
+
+    &__inline-message {
+        background-color: rgba(44, 154, 255, 0.08);
+        border-radius: 1.6rem;
+        padding: 1.6rem;
+        margin-bottom: 1.5rem;
+
+        @include mobile-or-tablet-screen {
+            margin-bottom: 0;
+        }
+
+        &__button {
+            padding: 0;
+            text-decoration: underline;
+            cursor: pointer;
+
+            &:hover {
+                background-color: transparent !important;
+
+                & > span {
+                    color: #0e0e0e !important;
+                }
+            }
+        }
+    }
```

Comment:
even though I used hideHoverStyles in the button, it didn't work 😅 so i had to manually disable it here